src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -175,28 +175,21 @@ export function DiscoverField({
   return (
     <>
       <EuiPopover
+        ownFocus
         display="block"
         button={
-          <EuiKeyboardAccessible>
-            <FieldButton
-              size="s"
-              className="dscSidebarItem"
-              isOpen={showDetails}
-              tabIndex={0}
-              onClick={() => {
-                togglePopover();
-              }}
-              onKeyPress={(event: any) => {
-                if (event.key === 'ENTER') {
-                  togglePopover();
-                }
-              }}
-              data-test-subj={`field-${field.name}-showDetails`}
-              fieldIcon={dscFieldIcon}
-              fieldAction={actionButton}
-              fieldName={fieldName}
-            />
-          </EuiKeyboardAccessible>
```

Comment:
I was able to get rid of `EuiKeyboardAccessible`, `tabIndex` and the `onKeyPress` by moving the internals of `FieldButton` to be inside an actual `<button>`